bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -7513,9 +7514,11 @@ private int updateDbDesign_041_to_042(final Connection conn, final SplashManager
             // Add new columns
             SQL.AddCol_BigInt(stmt, TABLE_TOUR_DATA, "tourDeviceTime_Recorded", DEFAULT_0);                            //$NON-NLS-1$
             SQL.AddCol_BigInt(stmt, TABLE_TOUR_DATA, "tourDeviceTime_Paused",   DEFAULT_0);                            //$NON-NLS-1$
+            SQL.AddCol_Float(stmt, TABLE_TOUR_DATA, "bodyFat",   DEFAULT_0);                            //$NON-NLS-1$
 
             SQL.RenameCol(stmt, TABLE_TOUR_DATA, "tourRecordingTime", "TourDeviceTime_Elapsed"); //$NON-NLS-1$ //$NON-NLS-2$
             SQL.RenameCol(stmt, TABLE_TOUR_DATA, "tourDrivingTime", "TourComputedTime_Moving"); //$NON-NLS-1$ //$NON-NLS-2$
+            SQL.RenameCol(stmt, TABLE_TOUR_DATA, "bikerWeight", "bodyWeight"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
**Fatal** issue which would cause lot of troubles when published:

`bikerWeight` is not renamed in `createTable_TourData()`

The problem would occure after the next version when MT is newly installed and `updateDbDesign_041_to_042()` is not run

Reply:
Thanks, I have fixed it in my current PR